solang-parser/src/solidity.lalrpop

```diff
@@ -185,6 +186,7 @@ ContractPart: ContractPart = {
     ModifierDefinition => ContractPart::FunctionDefinition(<>),
     ConstructorDefinition => ContractPart::FunctionDefinition(<>),
     TypeDefinition => ContractPart::TypeDefinition(<>),
+    Annotation => ContractPart::Annotation(<>),
```

Comment:
First of all the parser does not know that. Secondly we would like to reuse annotations for other features, for examples @xermicus suggested we use it for selector override rather than the other syntax we have.